src/utilities/findBreakingChanges.js

```diff
@@ -324,6 +324,30 @@ function findObjectTypeChanges(
   return schemaChanges;
 }
 
+function findInterfaceTypeChanges(
+  oldType: GraphQLInterfaceType,
+  newType: GraphQLInterfaceType,
+): Array<BreakingChange | DangerousChange> {
+  const schemaChanges = findFieldChanges(oldType, newType);
+  const interfacesDiff = diff(oldType.getInterfaces(), newType.getInterfaces());
+
+  for (const newInterface of interfacesDiff.added) {
+    schemaChanges.push({
+      type: DangerousChangeType.INTERFACE_ADDED_TO_OBJECT,
+      description: `${newInterface.name} added to interfaces implemented by ${oldType.name}.`,
+    });
+  }
+
+  for (const oldInterface of interfacesDiff.removed) {
+    schemaChanges.push({
+      type: BreakingChangeType.INTERFACE_REMOVED_FROM_OBJECT,
+      description: `${oldType.name} no longer implements interface ${oldInterface.name}.`,
+    });
+  }
```

Comment:
Please extract into a separate function and reuse for the object type, similar to `findFieldChanges`